packages/openapi-schema/schema.yml

```
@@ -2503,6 +2503,48 @@ paths:
                 $ref: '#/components/schemas/CheckSettingsFieldResponse'
+  /credit/recharge:
+    get:
+      tags:
+        - credit
+      summary: Get credit recharge
+      description: Get credit recharge
+      operationId: getCreditRecharge
+      responses:
+        '200':
+          description: successful operation
+          content: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)